src/Worker.Extensions.DurableTask/ManagedIdentityOptions.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask
+{
+    /// <summary>
+    /// Configuration options for Managed Identity.
+    /// This calss is the same as the one in WebJobs.Extensions.DurableTask/ManagedIdentityOptions.cs.
+    /// The implementation is kept in sync to ensure compatibility.
+    /// </summary>
+    public class ManagedIdentityOptions
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ManagedIdentityOptions"/> class.
+        /// </summary>
+        /// <param name="authorityHost">The host of the Azure Active Directory authority.</param>
+        /// <param name="tenantId">The tenant id of the user to authenticate.</param>
+        public ManagedIdentityOptions(Uri? authorityHost = null, string? tenantId = null)
+        {
+            this.AuthorityHost = authorityHost;
+            this.TenantId = tenantId;
+        }
+
+        /// <summary>
+        /// The host of the Azure Active Directory authority. The default is https://login.microsoftonline.com/.
```

Comment:
Azure Active Directory has been renamed to Entra ID.
```suggestion
        /// The host of the Entra ID authority. The default is https://login.microsoftonline.com/.
```